packages/core/src/codewhispererChat/controllers/chat/controller.ts

```diff
@@ -650,7 +651,9 @@ export class ChatController {
 
         const request = triggerPayloadToChatRequest(triggerPayload)
         const session = this.sessionStorage.getSession(tabID)
-        getLogger().info(`request from tab: ${tabID} conversationID: ${session.sessionIdentifier} request: %O`, request)
+        getLogger().info(
+            `request from tab: ${tabID} conversationID: ${session.sessionIdentifier} request: ${util.inspect(request, { depth: 12 })}`
```

Comment:
If we don't already have one, let's create our own `inspect()` function which wraps Node's util.inspect and uses our preferred parameters (depth=12 is probably too deep).

On web (non-node), our inspect() can use `partialClone` https://github.com/aws/aws-toolkit-vscode-staging/blob/70f9f1a41d140416912659107d8af91d34d5c464/packages/core/src/shared/utilities/collectionUtils.ts#L307 to limit depth and then pass it to e.g. JSON.stringify

Reply:
Sounds like a good idea. I named it `formatObj` and passed the object through `partialClone` either way so that we can provide more options for formatting even when not on web (Ex. key omission / replacement).